### PR TITLE
Addition of Note

### PR DESCRIPTION
**Pull request by joinimran**
As there is a clearance required on the page regarding office, I have researched and found that Office volume license version has few limitation and added this information as note. Please refer to the page https://docs.microsoft.com/en-us/deployoffice/delivery-optimization#:~:text=Volume%20licensed%20versions%20of%20Office,installing%20or%20for%20foreground%20updates. for the same.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8966

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/waas-delivery-optimization.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ For information about setting up Delivery Optimization, including tips for the b
     - Edge browser installs and updates
     - [Dynamic updates](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/the-benefits-of-windows-10-dynamic-update/ba-p/467847) 
 
+> [!NOTE]
+> Volume licensed versions of Office 2019, such as Office Professional Plus 2019, can take advantage of Delivery Optimization, but only for background updates. Delivery Optimization isn't available for installing or for foreground updates. The same applies to volume licensed versions of Project 2019 and Visio 2019.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:72]:
```suggestion
> Volume licensed versions of Office 2019, such as Office Professional Plus 2019, can take advantage of Delivery Optimization, but only for background updates. Delivery Optimization isn't available for installation or for foreground updates. The same applies to volume licensed versions of Project 2019 and Visio 2019.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9

**Review by illfated — APPROVED** [review]
Looks good to me. Thank you for the PR. 👍🏻

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**